### DISCUSSION TITLE
[Expander] Control GridLength of content and header

### DISCUSSION THREAD

**Discussion by sattew** [CommunityToolkit/Maui#1301] [New Feature Discussions]
Hi all.

I like this expander, but there are some things I can't do with it. For example. scrollview as content (see video below).
I want to suggest adding 2 new props `ContentHeight` and `HeaderHeight` to have more design options.

https://github.com/CommunityToolkit/Maui/assets/7888002/d14830ba-7f29-4c0c-a7a7-9d256cf989f4

Have no idea where the right place (issue or discusion) sorry for that.
Also it could be connected with several issues about Expander.

**Comment by VladislavAntonyuk** [comment]
header and content use the size of the content inside of them. you can set MaxHeight for the child control to achieve this result

**Reply by sattew** [reply]
But when you want to use all available height of parent(window/grid etc), you just cant use MaxHeight easily (or at all), am I right?

**Reply by VladislavAntonyuk** [reply]
ContentHeight and ContentWidth will not solve the issue. You need to set the size of your content. You can use DisplayInfo api to get the screen size.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
